schema/appState.json

```diff
@@ -0,0 +1,517 @@
+                "type": "boolean"
+            },
+            "error": {
+                "oneOf": [
+                    { "type": "null" },
+                    { "$ref": "#/definitions/WodinError" }
+                ]
+            }
+        },
+
+        "SensitivityScaleType": {
+            "type": "string",
+            "enum": ["Arithmetic", "Logarithmic"]
+        },
+
+        "SensitivityVariationType": {
+            "type": "string",
+            "enum": ["Percentage", "Range"]
+        },
+
+        "Batch": {
+            "type": "object",
+            "properties": {
+                "pars": {
+                    "$ref": "#/definitions/BatchPars"
+                },
+                "endTime": {
+                    "type": "number"
+                }
+            },
+            "required": ["pars", "endTime"],
+            "additionalProperties": false
+        },
+
+        "BatchPars": {
+            "type": "object",
+            "properties": {
+                "base": {
+                    "$ref": "#/definitions/OdinParameterValues"
+                },
+                "name": {
+                    "type": "string"
+                },
+                "values": {
+                    "$ref": "#/definitions/NumberArray"
+                }
+            },
+            "required": ["base", "name", "values"],
+            "additionalProperties": false
+        },
+
+        "FitDataState": {
+            "type": "object",
+            "properties": {
+                "data": {
+                    "$ref": "#/definitions/FitData"
+                },
+                "columns": {
+                    "oneOf": [
+                        { "type": "null" },
+                        { "$ref": "#/definitions/StringArray" }
+                    ]
+                },
+                "timeVariableCandidates": {
+                    "oneOf": [
+                        { "type": "null" },
+                        { "$ref": "#/definitions/StringArray" }
+                    ]
+                },
+                "timeVariable": {
+                    "type": ["null", "string"]
+                },
+                "linkedVariables": {
+                    "type": "object",
+                    "additionalProperties": {
+                        "type": ["null", "string"]
+                    }
+                },
+                "columnToFit": {
+                    "type": ["null", "string"]
+                },
+                "error": {
+                    "oneOf": [
+                        { "type": "null" },
+                        { "$ref": "#/definitions/WodinError" }
+                    ]
+                }
+            }
+        },
+
+        "FitData": {
+            "type": "object",
+            "additionalProperties": {
+                "$ref": "#/definitions/NumberArray"
+            }
+        },
+
+        "ModelFitState": {
+            "type": "object",
+            "properties": {
+                "fitting": {
```

Comment:
We shouldn't need this if we're not going to persist the state while it is fitting - this should always be false